mlflow/server/js/package.json

```diff
@@ -58,7 +58,7 @@
     "eslint-config-airbnb-base": "~12.0.0",
     "eslint-config-standard": "10.2.1",
     "eslint-import-resolver-webpack": "0.8.4",
-    "eslint-loader": "1.9.0",
+    "eslint-loader": "^2.1.2",
```

Comment:
Is this a necessary change?

Reply:
No sorry it was due to my local environment, just reverted this